### PR TITLE
feat(build): add ability to push to multiple registries from github actions

### PR DESCRIPTION
**Pull request by akhilerm**
**Why is this PR required? What issue does it fix?**:
Currently the multiarch images can be pushed only to a single registry without rebuilding.  Therefore the images are by default pushed to docker hub only. This is not ideal and images should be pushed to multiple registries. This PR fixes this issue by making use of the official docker [build-push-action](https://github.com/docker/build-push-action).

**What this PR does?**:
- make use of docker build-push-action
- push images to quay

**Does this PR require any upgrade changes?**:
No

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Any additional information for your reviewer?** : 
_Mention if this PR is part of any design or a continuation of previous PRs_
`QUAY_USERNAME` and `QUAY_TOKEN` secrets need to be added to the repo

**Checklist:**
- [ ] Fixes #<issue number>
- [x] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [x] Has the change log section been updated? 
- [ ] Commit has unit tests
- [ ] Commit has integration tests
- [ ] (Optional) Are upgrade changes included in this PR? If not, mention the issue/PR to track: 
- [ ] (Optional) If documentation changes are required, which issue on https://github.com/openebs/openebs-docs is used to track them:

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -132,9 +168,18 @@ jobs:
...
-        run: |
-          make docker.buildx.ndo
-          make buildx.push.ndo
+        uses: docker/build-push-action@v2
+        with:
+          file: ./build/ndm-operator/Dockerfile
+          push: true
+          platforms: linux/amd64, linux/arm64, linux/arm/v7, linux/ppc64le
+          tags: |
+            ${{ steps.docker_meta.outputs.tags }}
```
Comment (shubham14bajpai) [inline .github/workflows/release.yml:185]: nit: Newline

File: changelogs/unreleased/544-akhilerm [hunk 2]
```diff
@@ -0,0 +1 @@
+add support to push multiarch images to multiple registries
```
Comment (shubham14bajpai) [inline changelogs/unreleased/544-akhilerm:1]: nit: Newline

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -92,12 +103,21 @@ jobs:
...
-        run: |
-          make docker.buildx.ndm
-          make buildx.push.ndm
+        uses: docker/build-push-action@v2
+        with:
+          file: ./build/ndm-daemonset/Dockerfile
+          push: true
+          platforms: linux/amd64, linux/arm64, linux/arm/v7, linux/ppc64le
+          tags: |
+            ${{ steps.docker_meta.outputs.tags }}
```
Comment (shubham14bajpai) [inline .github/workflows/build.yml:120]: Should we pass the build-args as well? I see there are a few build-args in the Dockerfile

Comment (akhilerm) [reply .github/workflows/build.yml:124]: Done.

**Comment by xUnholy** [comment]
Appears to be CI is failing unit tests - any additional TODOs before this is ready for review?

**Comment by akhilerm** [comment]
> Appears to be CI is failing unit tests - any additional TODOs before this is ready for review?

@xUnholy @shubham14bajpai Will fix the unit tests, and mark it ready for review. Till then we will hold the review process.

**Comment by akhilerm** [comment]
@xUnholy @shubham14bajpai Fixed the issue with CI builds.This PR can now be reviewed

**Review by shubham14bajpai — APPROVED** [review]
LGTM

**Review by xUnholy — APPROVED** [review]
LGTM Nice changes!

**Review by kmova — APPROVED** [review]

**Pull request merged**